### ISSUE TITLE
Queries on having multiple variables in consequent of rules and weighted fuzzy rules

### ISSUE THREAD

**Issue by limgl1998** [aresio/simpful#12]
Hi. 
1. Is there any way I can have multiple variables in the consequent part of rules without duplicating the same rule several times?
Want:
IF (A IS A') THEN (B IS B') AND (C IS C')
Current implementation:
IF (A IS A') THEN (B IS B')
IF (A IS A') THEN (C IS C')
2. It was mentioned in your paper that one of the planned future releases includes weighted fuzzy rules. I'm wondering if there are any updates on it.
Thanks!

**Comment by sspola** [comment]
Hi @limgl1998 

thank you for you interest in simpful!

1. Unfortunately the library does not support multiple consequent statements at the moment. So you do have to specify different rules, like you point out. This was taken as a design choice during the first development, since it makes it easier to extend the support to other types of fuzzy systems in the future.
2. In one of my last commits (f9eb73b5c4a7617a472ab1bbeaba2fb28bfc418e) I tried to implement weights for Takagi-Sugeno models. The syntax is as follows:
           - "IF (Service IS poor) OR (Food IS rancid) THEN (Tip IS small) WEIGHT 0.5"
           - "IF (Service IS good) THEN (Tip IS average)  WEIGHT 1.0"
           - and so on...
I tested it myself, but if you are willing to try it out let me know if anything is wrong with it!
